modules/admin-portal/pages/gsql-output-file.adoc

```diff
@@ -1,36 +1,38 @@
 = Download GSQL Output File
 :sectnums:
 
-In a GSQL query, users can print any output from the query to a file. You can download the query output files from Admin Portal. 
+In a GSQL query, you can print any output from the query to a local file on the machine running TigerGraph. (See xref:gsql-ref:querying:output-statements-and-file-objects.adoc[] for details.)
+You can download these files from your web browser through the Admin Portal interface.
 
+This also applies to GraphStudio, because GraphStudio does not allow files to be saved from the web browser directly.
+Instead, any files saved through a query on GraphStudio must first be saved on the server, such as in a folder of your choice under `\home\tigergraph\`, then downloaded via Admin Portal.
```

Comment:
Is this true for on-prem GraphStudio? If you are running TigerGraph on-prem, your local machine *is* the server. 

If the file is saved on the server, then it is saved on your machine. 

I believe this limitation applies to TigerGraph Cloud only.